state-chain/pallets/cf-elections/src/electoral_systems/blockchain/delta_based_ingress.rs

```diff
@@ -242,7 +242,7 @@ where
 							Sink::on_ingress_reverted(
 								account.clone(),
```

Comment:
think decision was to just log here and remove this hook

Reply:
Now logging a warning instead.